bower.json

```diff
@@ -35,19 +37,21 @@
     "o-date": "^2.3.3",
     "o-errors": "^3.1.0",
     "o-expander": "^3.0.1",
+    "o-footer": "^4.1.3",
     "o-forms": "^2.0.0",
     "o-grid": "^4.2.0",
+    "o-header": "~6.0.3",
     "o-icons": "^4.0.0",
     "o-overlay": "^1.8.2",
     "o-typography": "^3.2.0",
     "o-viewport": "^2.0.0",
-    "superstore": "^2.1.0",
-    "superstore-sync": "^2.1.0",
-    "o-footer": "^4.1.3",
     "o-visual-effects": "^0.1.0",
-    "babel-polyfill-silencer": "^1.1.0"
+    "superstore": "^2.1.0",
+    "superstore-sync": "^2.1.0"
   },
   "resolutions": {
-    "o-viewport": "^2.0.0"
+    "o-viewport": "^2.0.0",
```

Comment:
could you try deleting these resolutions, then a `rm -rf bower_components && bower install`. They are probably not needed any more